processor/internal/transformer/destination_transformer/embedded/warehouse/events.go

```diff
@@ -236,9 +236,7 @@ func (t *Transformer) extractCommonProps(tec *transformEventContext) (map[string
 	if err = setDataAndMetadataFromRules(tec, commonData, commonMetadata, rules.ExtractRules); err != nil {
 		return nil, nil, fmt.Errorf("extract: setting data and column types from rules: %w", err)
 	}
-
-	eventName, _ = commonData[eventColName].(string)
-	if utils.IsEmptyString(eventName) {
+	if len(strings.TrimSpace(utils.ToString(commonData[eventColName]))) == 0 {
```

Comment:
Is the change in this file only about trimming? If yes, can include trimming inside `IsEmptyString` function instead of using `ToString`?

Reply:
If we move this to `IsEmptyString`, its implementation for JS would then be different as the empty-spaced string would result in `false` in case of JavaScript. For now keeping it like this.
```
//
//	const isBlank = (value) => {
//	 try {
//	   return _.isEmpty(_.toString(value));
//	 } catch (e) {
//	   console.error(`Error in isBlank: ${e.message}`);
//	   return false;
//	 }
//	};
```